README.md

```diff
@@ -168,6 +168,12 @@ A variable declaration declares the required and optional variables for this que
 #### Symbol
 - `:`
 
+### Nullable variable declaration
+Declare if the variable can accepts `null` values. It is only allowed inside a query declaration.
+
+#### Symbol
+- `?`
```

Comment:
Wrong place. This section is for the statement types, i.e. the symbol immediately following `-- #`. `?` is a modifier to the argument type, not a statement type.